### PR TITLE
Fix preset config files.

### PR DESCRIPTION
**Pull request by hansieodendaal**
## Description
Updated the preset config files to be consistent:
- Example config (`tari_config_example.toml`) [renamed to eliminate confusion, replaced deleted content, removed `mainnet` settings]
- Windows config (`windows.toml`) [removed `mainnet` settings, aligned with example config]
- Minimal sample config (`tari-sample.toml`) [aligned ordering with example config]

## Motivation and Context
Some information was outdated in these files and a large portion of the example file was deleted recently by accident.

## How Has This Been Tested?
Tested by starting a base node in Windows

## Types of changes

* [X] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [X] Documentation

## Checklist:

* [X] I'm merging against the `development` branch.
* [X] I ran `cargo-fmt --all` before pushing.
* [X] I ran `cargo test` successfully before submitting my PR.
* [X] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: common/config/tari_config_sample.toml [hunk 1]
```diff
@@ -1,14 +1,167 @@
...
 network = "ridcully"
 
 
+# Configuration options for testnet
+
 [base_node.ridcully]
+# The type of database backend to use. Currently supported options are "memory" and "lmdb". LMDB is recommnded for
+# almost all use cases.
 db_type = "lmdb"
-transport = "tor"
```
Comment (SWvheerden) [inline common/config/tari_config_sample.toml:11]: Why is the default tor removed

File: common/config/tari_config_sample.toml [hunk 2]
```diff
@@ -19,14 +172,307 @@ peer_seeds = [
     #tbn-sydney
     "ac7fba427913a653a27b69c05549e14d9e87cb7849ea0c740d6c8a5855a3882a::/onion3/avteohvrvvfy4fff7ona2wy6yhb7i4ss4aklp74v64knofvev3vd5yyd:18141",
 ]
-enable_mining = false
-grpc_enabled = true
-grpc_address = "127.0.0.1:18142"
-grpc_wallet_address = "127.0.0.1:18143"
```
Comment (SWvheerden) [inline common/config/tari_config_sample.toml:25]: I think this needs to be in.

File: common/config/tari_config_sample.toml [hunk 3]
```diff
@@ -19,14 +172,307 @@ peer_seeds = [
...
+#reorg_pool_storage_capacity = 10_000
+#reorg_tx_ttl = 300
+
+# The maximum number of transactions that can be skipped when compiling a set of highest priority transactions,
+# skipping over large transactions are performed in an attempt to fit more transactions into the remaining space.
+# This parameter only affects mining nodes. You can ignore it if you are only running a base node. Even so, changing
+# this parameter should not affect profitabilty in any meaningful way, since the transaction weights are selected to
+# closely mirror how much block space they take up
+#weight_tx_skip_count = 20
+
```
Comment (SWvheerden) [inline common/config/tari_config_sample.toml]: Does this even work?
There is no testnet network.

File: common/config/tari_config_sample.toml [hunk 4]
```diff
@@ -19,14 +172,307 @@ peer_seeds = [
...
+#tor_identity_file = "./tor.json" # or ".\\tor.json"
+
+# A path to the file that stores the wallet's tor hidden service private key, if using the tor transport.
+#wallet_tor_identity_file = "./wallet-tor.json" # or ".\\wallet-tor.json"
+
+########################################################################################################################
+#                                                                                                                      #
+#                                             Mempool Configuration Options                                            #
+#                                                                                                                      #
+########################################################################################################################
```
Comment (SWvheerden) [inline common/config/tari_config_sample.toml]: We can remove this.  As Mainnet panics.

File: common/config/tari_config_sample.toml [hunk 5]
```diff
@@ -19,14 +172,307 @@ peer_seeds = [
...
+#reorg_pool_storage_capacity = 10_000
+#reorg_tx_ttl = 300
+
+# The maximum number of transactions that can be skipped when compiling a set of highest priority transactions,
+# skipping over large transactions are performed in an attempt to fit more transactions into the remaining space.
+# This parameter only affects mining nodes. You can ignore it if you are only running a base node. Even so, changing
+# this parameter should not affect profitabilty in any meaningful way, since the transaction weights are selected to
+# closely mirror how much block space they take up
+#weight_tx_skip_count = 20
+
```
Comment (SWvheerden) [inline common/config/tari_config_sample.toml]: We can remove hence mainnet config.

**Review by SWvheerden — CHANGES_REQUESTED** [review]
Except for the tor transport option this looks fine. 
I left comments. The previous PR on the toml files had a goal of reducing the config. This reverts most of that. 

There is also another 3rd config file that is now mis matched to these two. 
All 3 config files should match or they should be removed to only 2. Currently the options between the two dont matchas well.

File: common/config/tari_config_sample.toml (hunk 1 above)
Comment (hansieodendaal) [reply common/config/tari_config_sample.toml:11]: It is still in the file not removed, just in the proper place. See line 222

File: common/config/tari_config_sample.toml (hunk 2 above)
Comment (hansieodendaal) [reply common/config/tari_config_sample.toml:25]: Yes, it is still in. See lines 194, 197 & 200.

File: common/config/tari_config_sample.toml (hunk 3 above)
Comment (hansieodendaal) [reply common/config/tari_config_sample.toml]: Good pickup! Yes, changed.

File: common/config/tari_config_sample.toml (hunk 4 above)
Comment (hansieodendaal) [reply common/config/tari_config_sample.toml]: Agreed, removed

File: common/config/tari_config_sample.toml (hunk 5 above)
Comment (hansieodendaal) [reply common/config/tari_config_sample.toml]: Agreed, removed

**Review by stringhandler — APPROVED** [review]
I think we need to consolidate our approach to config files. Until then, this can go in as is

**Pull request merged**